### DISCUSSION TITLE
umi4新建项目mpa开启报错

### DISCUSSION THREAD

**Discussion by dengrujie** [umijs/umi#9096] [Q&A] [answered]
通过pnpm新建umi4的项目，在.umirc.ts项目里配置mpa，运行dev报错Invalid config keys: mpa

**Comment by fz6m** [comment, marked as answer]
今天晚上发版后再尝试，现在还没发布。

**Reply by dengrujie** [reply]
更新版本后，mpa模式的配置是怎么配置呢，直接赋值空对象会提示webpack的entry错误，添加entry属性又提示entry是不允许赋值的

**Reply by fz6m** [reply]
https://umijs.org/docs/guides/mpa#mpa-%E6%A8%A1%E5%BC%8F
